contracts/modules/assessment/Assessment.sol

```diff
@@ -132,25 +132,49 @@ contract Assessment is IAssessment, MasterAwareV2 {
   /// @param to      The member address where the NXM is transfered to. Useful for switching
   ///                membership during stake lockup period and thus allowing the user to withdraw
   ///                their staked amount to the new address when possible.
-  function unstake(uint96 amount, address to) external whenNotPaused override {
+  function unstake(uint96 amount, address to) external whenNotPaused {
```

Comment:
do we want to keep the `override` modifier here?

Reply:
Yes, good catch. Both unstake and unstakeFor should have override modifiers